Sources/MapboxNavigation/ImageCache.swift

```diff
@@ -1,11 +1,23 @@
 import UIKit
 
+internal class ImageMemory: NSObject, NSDiscardableContent {
+    func beginContentAccess() -> Bool { return true }
+    func endContentAccess() { }
+    func discardContentIfPossible() {}
```

Comment:
Doesn't that result in images data being never evicted? Should we still clear this cache at some point? For example after app was shut down or after some disk quota exceeded?

Reply:
We clear the memory when the `style` change or when `traitCollection` change. because it's used in `WayNameView` and `InstructionLabel` right now. The `NSDiscardableContent` only prevents the cached content removing when App goes to the background. But because we don't store it to the disk. When the App is shut down, it will be cleared as well.